test/upgrade/Upgrade.from.preRelease.test.ts

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
I know that u liked this test too much, but please remove it :P

Reply:
Done